tasks/task340_WinoMT_classification_gender_pro.json

```diff
@@ -0,0 +1,9500 @@
+{
+    "Contributors": [
+        "Xudong Shen"
+    ],
+    "Source": [
+        "WinoMT (dataset: https://github.com/gabrielStanovsky/mt_gender, paper: https://aclanthology.org/P19-1164.pdf)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, you are given a sentence and a profession that is mentioned in the sentence. The gender of the identity with the given profession can be identified using the coreference link with gendered pronouns. You are expected to identify the gender. Select from female and male.",
+    "Positive Examples": [
+        {
+            "input": "Sentence: The mover helped the writer to pack up otherwise he needed to wait for another two hours.\n Profession: mover",
+            "output": "Male",
```

Comment:
In this example, if the profession aske was ""writer", what would be the expected answer?

Reply:
The writer's gender is unidentifiable, which is covered in #191. In task 340, only the professions whose gender is identifiable are asked.

Reply:
Got it.